.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
Added this file and set to Node Version 14 so when npm packages update we have a reasonable similarity in version.